tests/python/contrib/test_ethosn/test_addition.py

```diff
@@ -173,6 +173,61 @@ def test_addition_both_inputs_constants(
     assert "qnn.add" not in mod.astext(False)
 
 
+@requires_ethosn
+@pytest.mark.parametrize("dtype", ["uint8", "int8"])
+@pytest.mark.parametrize(
+    "lhs_shape,lhs_is_constant,rhs_shape,rhs_is_constant",
+    [
+        ((1, 4, 4, 8), False, (1, 4, 4, 8), True),
+        ((1, 16, 12, 4), True, (1, 16, 12, 4), False),
```

Comment:
It might also be good to test broadcast shapes not covered by `qnn_add_to_depthwise` e.g. ((1, 4, 4, 8), False, (1, 1, 4, 8), True), I believe this should be supported?

Reply:
Discussed this offline - it's not something that's currently supported by the support library